packages/keychain/src/hooks/account.ts

```diff
@@ -238,3 +239,11 @@ export function useAccount() {
     };
   }, [controller]);
 }
+
+export function useUsername(address: string) {
```

Comment:
It doesnt cover other name providers, such as `.stark`
Maybe acceptable until we merge both iframes, need to validate it with @clicksave 

| Profile | Keychain |
|:-:|:-:|
![image](https://github.com/user-attachments/assets/1cbe632f-f056-4cd9-bcce-76d1b7b5419e) | ![image](https://github.com/user-attachments/assets/ffec37a5-b69a-4585-bf96-bf0ca405e47d)